src/yder.c

```diff
@@ -62,16 +64,17 @@ static void y_write_log_console(const char * app_name, const time_t date, const
       break;
   }
   if (level & Y_LOG_LEVEL_WARNING || level & Y_LOG_LEVEL_ERROR) {
-    // Write to stderr
+    /* Write to stderr */
     output = stderr;
   } else {
-    // Write to stdout
+    /* Write to stdout */
     output = stdout;
   }
   fprintf(output, "%s - %s %s: %s\n", date_stamp, app_name, level_name, message);
   fflush(output);
 }
 
+#ifndef _WIN32
```

Comment:
I imagine you disable syslog because there's no equivalent in Windows, but I see a problem in that, a program wouldn't compile and the reason would be cryptic.

Instead of disabling `y_write_log_syslog I'd rather keep it but empty, or redirect to `y_write_log_console`, maybe the redirection is the best in this case, so the application would still have logs.

Reply:
What do you thin about using [event logging on Windows](https://www.youtube.com/watch?v=J6vUOyxmU1o)? :-) I've used it a lot for logging services and it is the closest option for *nix syslog.

Reply:
Writing on Windows Event Log is better than just disabling syslog I think, I'd prefer that :)